idaes/models_extra/power_generation/costing/power_plant_capcost.py

```diff
@@ -0,0 +1,1995 @@
+                )
+                * 1
+                * pyunits.year  # variable costs for 1 year
+                + (
+                    land_cost * land_cost_units_factor
+                    if land_cost is not None
+                    else 0 * CE_index_units
+                )
+                + self.total_TPC
+                * self.pct_TPC  # other owners costs (other + spare parts + financing + other pre-production)
+            )
+
+            self.tasc_toc_factor = Param(
+                initialize=1.093,
+                mutable=True,
+                doc="TASC/TOC factor from Exhibit 3-7 reference 1, real for three years 1.093, real for five years 1.154, nominal for three years 1.242, nominal for five years 1.289",
+            )
+
+            self.total_as_spent_cost = Expression(
+                expr=self.total_overnight_capital * self.tasc_toc_factor
+            )
+
+            self.fixed_charge_factor = Param(
+                initialize=0.0707,
+                mutable=True,
+                doc="Fixed charge rate from Exhibit 3-5 based on CRF values, real for three/five years 0.0707, nominal for three/five years 0.0886",
+            )
+            self.annualized_cost = Expression(
+                expr=self.fixed_charge_factor * self.total_as_spent_cost
+            )
+
+            self.add_COE_cost = Param(
+                initialize=70.9e-6,
+                mutable=True,
+                doc="additional cost to be added to the COE calculations in millions",
+                units=CE_index_units / pyunits.MWh,
+            )
+
+            # only build COE for power plants
+            if net_power is not None:
+                self.cost_of_electricity = Expression(
+                    expr=(
+                        (
+                            self.annualized_cost / pyunits.year
+                            + self.total_fixed_OM_cost / pyunits.year
+                            + self.total_variable_OM_cost[0] * self.capacity_factor
+                        )
+                        / (
+                            self.capacity_factor
+                            * self.net_power[0]
+                            * 8760
+                            * pyunits.hr
+                            / pyunits.year
+                        )
+                        + self.add_COE_cost
+                    )
+                )
+            if tonne_CO2_capture is not None:
+                if not hasattr(self, "tonne_CO2_capture"):
+                    self.tonne_CO2_capture = Param(
+                        initialize=tonne_CO2_capture, mutable=True, units=pyunits.ton
+                    )
+                self.cost_of_capture = Expression(
+                    expr=(
+                        (
+                            self.annualized_cost / pyunits.year
+                            + self.total_fixed_OM_cost / pyunits.year
+                            + self.total_variable_OM_cost[0] * self.capacity_factor
```

Comment:
Same comment in regards to multiplying the `total_variable_OM_cost` by the `capacity_factor`. Double check this